com.github.reduz.ChibiTracker.yaml

```diff
@@ -0,0 +1,44 @@
+app-id: com.github.reduz.ChibiTracker
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: chibitracker
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --filesystem=host
+
+modules:
+  - shared-modules/SDL/SDL-1.2.15.json
+  - name: scons
+    buildsystem: simple
+    cleanup: '*'
+    build-commands:
+      - python setup.py install --prefix=/app
+    sources:
+      - type: archive
+        url: http://prdownloads.sourceforge.net/scons/scons-3.0.1.tar.gz
+        sha256: 24475e38d39c19683bc88054524df018fe6949d70fbd4c69e298d39a0269f173
+
+  - name: chibitracker
+    buildsystem: simple
+    build-commands:
+      - scons
+      - scons install prefix=/app
+      - install -Dm644 com.github.reduz.ChibiTracker.appdata.xml /app/share/appdata/com.github.reduz.ChibiTracker.appdata.xml
+      - install -Dm644 com.github.reduz.ChibiTracker.desktop /app/share/applications/com.github.reduz.ChibiTracker.desktop
+      - install -Dm644 program/cticon.png /app/share/icons/hicolor/32x32/apps/com.github.reduz.ChibiTracker.png
```

Comment:
Having only a 32px size is not enough. The icon will look blurry :( you can ship one size if you have an svg version of it.

Reply:
Actually, I searched quite a bit and there doesn't seem to be a bigger version of the icon on the internet. =(
However, that icon looks fine on the my desktop. It looks fine on Arc-Menu, the GNOME App Launcher and Dash to Panel.

![screenshot from 2018-11-16 13-18-54](https://user-images.githubusercontent.com/24463334/48630078-71526880-e9a2-11e8-98e0-523af672a581.png)
![screenshot from 2018-11-16 13-19-10](https://user-images.githubusercontent.com/24463334/48630079-71526880-e9a2-11e8-8871-4731215944d8.png)
![screenshot from 2018-11-16 13-19-27](https://user-images.githubusercontent.com/24463334/48630080-71eaff00-e9a2-11e8-8179-6823b9f0e4d9.png)

It looks reasonably fine on my FullHD screen.